main/cogs/ladder.py

```diff
@@ -40,7 +63,184 @@ async def ladder(self, ctx, mode="off"):
                     message = "```"
             message += "```"
 
-            await ctx.send(message)
+            if message != "``````":
+                await ctx.send(message)
+        else:
+            embed = discord.Embed(color=0xEA7D07)
+            embed.add_field(name='The !ladder command must be used here:', value=f'<#{ev.get_var("bot_spam_channel_id")}>')
+            await ctx.send(embed=embed)
+
+    @commands.command(name="compactLadder", help="Display the ladder in a compact view [mode] [min]")
+    @commands.cooldown(1, 60, commands.BucketType.default)
+    async def compact_ladder(self, ctx, mode="off", min_games=0):
```

Comment:
My maybe hot take would be that I would rather have the ladder be a couple characters misaligned for SuleimantheLawgiver than make all output wider just for the very rare extremely long username
Compact is basically just the same ladder but with win % removed. So that makes me think is there something else we could do to adjust the spacing of the main ladder display and make it unnecessary? Remove a decimal place from win %? Adjust something else about the display? Etc
OR - are there more things we could ADD to the default ladder command to make it more interesting/meaningful to people who want a more full display
That might be the better approach to keep both commands meaningful and give people the ladder display they are looking for

Reply:
yeah, this is a good thought. gonna combine it with what was discussed in the discord about capping the length for usernames.